<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ChromeTest"
           tests="4"
           failures="4"
           errors="0"
           skipped="0"
           time="250.643">
  <properties />
            <testcase name="registerNewCustomerSubysystemPage" time="128.442" classname="automationFramework.TestCases.CreateUnregistredCustomerTest">
                <failure type="org.openqa.selenium.NoSuchElementException"
                  message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//*[@id=&apos;customerContacts_list_tbl&apos;]/div/div[1]/table/tbody/tr/td[1]&quot;}
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.28.455520 (cc17746adff54984afff480136733114c6b3704b),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.04 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &apos;3.0.1&apos;, revision: &apos;1969d75&apos;, time: &apos;2016-10-18 09:49:13 -0700&apos;
System info: host: &apos;NACTS-081729&apos;, ip: &apos;149.63.89.30&apos;, os.name: &apos;Windows 7&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.1&apos;, java.version: &apos;1.8.0_77&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.28.455520 (cc17746adff54984afff480136733114c6b3704b), userDataDir=C:\Users\200123\AppData\Local\Temp\scoped_dir27888_5689}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=57.0.2987.133, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 2fa1f30e6b9e5096f2dfa29a47eb9626
*** Element info: {Using=xpath, value=//*[@id=&apos;customerContacts_list_tbl&apos;]/div/div[1]/table/tbody/tr/td[1]}"
              >
<![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='customerContacts_list_tbl']/div/div[1]/table/tbody/tr/td[1]"}
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.28.455520 (cc17746adff54984afff480136733114c6b3704b),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.04 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'NACTS-081729', ip: '149.63.89.30', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_77'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.28.455520 (cc17746adff54984afff480136733114c6b3704b), userDataDir=C:\Users\200123\AppData\Local\Temp\scoped_dir27888_5689}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=57.0.2987.133, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 2fa1f30e6b9e5096f2dfa29a47eb9626
*** Element info: {Using=xpath, value=//*[@id='customerContacts_list_tbl']/div/div[1]/table/tbody/tr/td[1]}
  sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
  sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
  java.lang.reflect.Constructor.newInstance(Unknown Source)
  org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
  org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)
  org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
  org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
  org.openqa.selenium.By$ByXPath.findElement(By.java:361)
  org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:360)
  automationFramework.PageObjects.NewCustomerDisplayPage.getFname(NewCustomerDisplayPage.java:35)
  automationFramework.TestCases.CreateUnregistredCustomerTest.verifyNewCustomerCreation(CreateUnregistredCustomerTest.java:179)
  automationFramework.TestCases.CreateUnregistredCustomerTest.registerNewCustomerSubysystemPage(CreateUnregistredCustomerTest.java:168)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
      </failure>

          </testcase>
            <testcase name="registerNewCustomerNoRecordFound" time="49.059" classname="automationFramework.TestCases.CreateUnregistredCustomerTest">
                <failure type="java.lang.AssertionError"
                  message="expected [5472646174] but found [5472646174,undefined,undefined]"
              >
<![CDATA[java.lang.AssertionError: expected [5472646174] but found [5472646174,undefined,undefined]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.CreateUnregistredCustomerTest.verifyNewCustomerCreation(CreateUnregistredCustomerTest.java:185)
  automationFramework.TestCases.CreateUnregistredCustomerTest.registerNewCustomerNoRecordFound(CreateUnregistredCustomerTest.java:63)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
      </failure>

          </testcase>
            <testcase name="registerNewCustomerInvalidToken" time="19.883" classname="automationFramework.TestCases.CreateUnregistredCustomerTest">
                <failure type="java.lang.AssertionError"
                  message="expected [7766141175] but found [7766141175,undefined,undefined]"
              >
<![CDATA[java.lang.AssertionError: expected [7766141175] but found [7766141175,undefined,undefined]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.CreateUnregistredCustomerTest.verifyNewCustomerCreation(CreateUnregistredCustomerTest.java:185)
  automationFramework.TestCases.CreateUnregistredCustomerTest.registerNewCustomerInvalidToken(CreateUnregistredCustomerTest.java:130)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
      </failure>

          </testcase>
            <testcase name="registerNewCustomerTranistTokenFound" time="53.259" classname="automationFramework.TestCases.CreateUnregistredCustomerTest">
                <failure type="java.lang.AssertionError"
                  message="expected [6610372263] but found [6610372263,undefined,undefined]"
              >
<![CDATA[java.lang.AssertionError: expected [6610372263] but found [6610372263,undefined,undefined]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.CreateUnregistredCustomerTest.verifyNewCustomerCreation(CreateUnregistredCustomerTest.java:185)
  automationFramework.TestCases.CreateUnregistredCustomerTest.registerNewCustomerTranistTokenFound(CreateUnregistredCustomerTest.java:104)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
